Formula/nb.rb

```diff
@@ -0,0 +1,38 @@
+class Nb < Formula
+  desc "Command-line and local web note‑taking, bookmarking, and archiving"
+  homepage "https://xwmx.github.io/nb"
+  url "https://github.com/xwmx/nb/archive/refs/tags/6.11.1.tar.gz"
+  sha256 "14627160974128d32398de3115b22728f236a62b51178c226bfbc35565f0b186"
+  license "AGPL-3.0-or-later"
+  head "https://github.com/xwmx/nb.git", branch: "master"
+
+  depends_on "nano" => :test
+  depends_on "bat"
+  depends_on "nmap"
+  depends_on "pandoc"
+  depends_on "ripgrep"
+  depends_on "tig"
+  depends_on "w3m"
+
+  uses_from_macos "bash"
+
+  def install
+    bin.install "nb", "bin/bookmark"
+
+    bash_completion.install "etc/nb-completion.bash" => "nb.bash"
+    zsh_completion.install "etc/nb-completion.zsh" => "_nb"
+    fish_completion.install "etc/nb-completion.fish" => "nb.fish"
+  end
+
+  test do
+    ENV["EDITOR"] = which "nano"
+
```

Comment:
Why nano?

Reply:
How about an empty string?

Reply:
That might work since the editor is never actually opened during any of the tests. I'll try that.

Reply:
Okay I just tried it and the build failed. I'm going to see if vim is installed be default on Ubuntu, but if not, what do you want to do?

Reply:
I just tried using a non-empty placeholder and it works! `nano` is now not required.